diesel_tests/tests/combination.rs

```diff
@@ -57,7 +57,7 @@ fn union_all() {
     let data: Vec<_> = users
         .filter(id.le(tess.id))
         .union_all(users.filter(id.ge(tess.id)))
-        .positional_order_by(2) // name is the second column
+        .positional_order_by(OrderColumn::from(2)) // name is the second column
```

Comment:
That shouldn't be required anymore now that you've implemented `IntoPositionalOrderExpr`. Given that I would prefer to roll back these changes (`OrderColumn::from()`) here.

Reply:
sorry, I wanted to implement IntoPositionalOrderExpr for tuples too but then got busy!